### ISSUE TITLE
[BUG] Starting from 5.2.2, TiddlyWiki fails to correctly build TiddlyWiki Classic

### ISSUE THREAD

**Issue by YakovL** [Jermolene/TiddlyWiki5#7932]
### Describe the bug

The title sums this up, see also steps to reproduce. See an example of a broken at https://groups.google.com/g/tiddlywikiclassic/c/Yv9xa0BYu2s/m/WIEmRxWLBQAJ

### Expected behavior

Building with 5.2.2 should work as with 5.2.1

### To Reproduce

1. Clone [TiddlyWikiClassic](https://github.com/TiddlyWiki/TiddlyWikiClassic/);
2. Run `npm i` and `npm run build-core`;
   - it works: open the resulting html, looks fine
3. In package.json, substitute `"tiddlywiki": "5.2.1"` with `"tiddlywiki": "^5.2.1"` or just `"tiddlywiki": "5.2.2"`; remove `node_modules`;
4. Repeat step 2;
   - the result is a broken TW.

### TiddlyWiki Configuration

Reproduced on Windows 10 (and probably on Unix for Okido, not quite sure); haven't investigated if depends on the node.js version yet

### Additional context

I haven't done a deeper investigation yet, this is just an initial report

**Comment by Jermolene** [comment]
Thank you @YakovL I'm investigating now.

**Comment by Jermolene** [comment]
Hi @YakovL I've made a tentative fix in #7933. You should be able to try it out by modifying your package.json to reference the branch of the PR:

```
...
  "dependencies": {
	"tiddlywiki": "Jermolene/TiddlyWiki5#fix-classictools"
  },
...
```

Let me know how you get on.

**Comment by YakovL** [comment]
Thanks, @Jermolene, looks like the fixed version works fine (produces the same result as 5.2.1, at least on the current dev snapshot)!

**Comment by Jermolene** [comment]
Thanks @YakovL. I'll merge the fix shortly into the prerelease for v5.3.4. At that point you will be able to use it as `Jermolene/TiddlyWiki5#master`. Perhaps it would make sense for you to stick with master rather than a specified version to make sure that we don't miss any further future incompatibilities.

**Comment by saqimtiaz** [comment]
@Jermolene @YakovL this issue has been resolved and the issue can be closed.
